bin/metadata.js

```diff
@@ -3,36 +3,57 @@
 
 const fs = require('fs');
 const path = require('path');
+const argv = require('yargs')
+  .usage('$0 <identifier> [options]', 'Retrieves metadata for a PR and validates them against nodejs/node PR rules')
+  .detectLocale(false)
+  .demandCommand(1, 'Pull request identifier (id or URL) is required as first argument.')
+  .option('o', {
+    alias: 'owner',
+    demandOption: false,
+    default: 'nodejs',
+    describe: 'GitHub owner of the PR repository',
+    type: 'string'
+  })
+  .option('r', {
+    alias: 'repo',
+    demandOption: false,
+    default: 'node',
+    describe: 'GitHub repository of the PR',
+    type: 'string'
+  })
+  .help('h')
+  .alias('h', 'help').argv;
 
 function loadQuery(file) {
   const filePath = path.resolve(__dirname, '..', 'queries', `${file}.gql`);
   return fs.readFileSync(filePath, 'utf8');
 }
+
 const PR_QUERY = loadQuery('PR');
 const REVIEWS_QUERY = loadQuery('Reviews');
 const COMMENTS_QUERY = loadQuery('PRComments');
 const COMMITS_QUERY = loadQuery('PRCommits');
 const USER_QUERY = loadQuery('User');
 
-const { request, requestAll } = require('../lib/request');
-const { getCollaborators } = require('../lib/collaborators');
+const {request, requestAll} = require('../lib/request');
+const {getCollaborators} = require('../lib/collaborators');
 const logger = require('../lib/logger');
-const { ReviewAnalyzer } = require('../lib/reviews');
+const {ReviewAnalyzer} = require('../lib/reviews');
 const PRChecker = require('../lib/pr_checker');
 const MetadataGenerator = require('../lib/metadata_gen');
 
 // const REFERENCE_RE = /referenced this pull request in/
 
-const PR_ID = parsePRId(process.argv[2]);
-const OWNER = process.argv[3] || 'nodejs';
-const REPO = process.argv[4] || 'node';
+let OWNER;
+let REPO;
+const PR_ID = parsePRId(argv._[0]);
```

Comment:
This can be just `argv.identifier`

Reply:
Yup, got it at the same time, I changed the yargs.usage last minute, my bad